wkcuber/README.md

```diff
@@ -1,155 +1,49 @@
-[![Build Status](https://img.shields.io/github/actions/workflow/status/scalableminds/webknossos-libs/.github/workflows/ci.yml?branch=master)](https://github.com/scalableminds/webknossos-libs/actions?query=workflow%3A%22CI%22)
-[![Documentation](https://img.shields.io/badge/docs-passing-brightgreen.svg)](https://docs.webknossos.org/wkcuber/index.html)
-[![Code Style](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
+# wkcuber
 
-Python library for creating and working with [WEBKNOSSOS](https://webknossos.org) [WKW](https://github.com/scalableminds/webknossos-wrap) datasets. WKW is a container format for efficiently storing large, scale 3D image data as found in (electron) microscopy.
-
-The tools are modular components to allow easy integration into existing pipelines and workflows.
+Python CLI for creating and working with [WEBKNOSSOS](https://webknossos.org/) [WKW](https://github.com/scalableminds/webknossos-wrap) datasets. WKW is a container format for efficiently storing large, scale 3D image data as found in (electron) microscopy.
 
 ## Features
 
-* `wkcuber`: Convert supported input files to fully ready WKW datasets (includes type detection, downsampling, compressing and metadata generation)
-* `wkcuber.convert_image_stack_to_wkw`: Convert image stacks to fully ready WKW datasets (includes downsampling, compressing and metadata generation)
-* `wkcuber.export_wkw_as_tiff`: Convert WKW datasets to a tiff stack (writing as tiles to a `z/y/x.tiff` folder structure is also supported)
-* `wkcuber.cubing`: Convert image stacks (e.g., `tiff`, `jpg`, `png`, `bmp`, `dm3`, `dm4`) to WKW cubes
-* `wkcuber.tile_cubing`: Convert tiled image stacks (e.g. in `z/y/x.ext` folder structure) to WKW cubes
-* `wkcuber.convert_knossos`: Convert KNOSSOS cubes to WKW cubes
-* `wkcuber.convert_nifti`: Convert NIFTI files to WKW files (Currently without applying transformations).
-* `wkcuber.convert_raw`: Convert RAW binary data (.raw, .vol) files to WKW datasets
-* `wkcuber.downsampling`: Create downsampled magnifications (with `median`, `mode` and linear interpolation modes). Downsampling compresses the new magnifications by default (disable via `--no_compress`).
-* `wkcuber.compress`: Compress WKW cubes for efficient file storage (especially useful for segmentation data)
-* `wkcuber.metadata`: Create (or refresh) metadata (with guessing of most parameters)
-* `wkcuber.recubing`: Read existing WKW cubes in and write them again specifying the WKW file length. Useful when dataset was written e.g. with file length 1.
-* `wkcuber.check_equality`: Compare two WKW datasets to check whether they are equal (e.g., after compressing a dataset, this task can be useful to double-check that the compressed dataset contains the same data).
-* Most modules support multiprocessing
+wkcuber offers some commands to work with WEBKNOSSOS datasets:
+
+- `wkcuber compress`: Compress a WEBKNOSSOS dataset
```

Comment:
@normanrz  More general question: Do we want to keep the `wkcuber` name for the command or switch to just `webknossos <command>`?